aws/rds_cluster_snapshots_types.go

```diff
@@ -0,0 +1,60 @@
+package aws
+
+import (
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+)
+
+type DBClusterSnapshots struct {
+	SnapshotNames []string
+}
+
+// Name of the AWS resource
+func (snapshot DBClusterSnapshots) ResourceName() string {
+	return "rdssnapshots"
+}
+
+// Names of the RDS DB Cluster Snapshots
+func (snapshot DBClusterSnapshots) ResourceIdentifiers() []string {
+	return snapshot.SnapshotNames
+}
+
+// MaxBatchSize decides how many cluster snapshots to delete in one call.
+func (snapshot DBClusterSnapshots) MaxBatchSize() int {
+	return 200
+}
+
+// Nuke/Delete all RDS DB Cluster snapshots
+func (snapshot DBClusterSnapshots) Nuke(session *session.Session, identifiers []string) error {
+	if err := nukeAllRdsClusterSnapshots(session, awsgo.StringSlice(identifiers)); err != nil {
+		return errors.WithStackTrace(err)
+	}
+
+	return nil
+}
+
+type RdsClusterSnapshotDeleteError struct {
+	name string
+}
+
+func (e RdsClusterSnapshotDeleteError) Error() string {
+	return "RDS DB Cluster Snapshot:" + e.name + "was not deleted"
```

Comment:
Spaces around the name are missing. It's more idiomatic to use `fmt.Sprintf` for this sort of thing, which helps avoid this sort of error:

```suggestion
	return fmt.Sprintf("RDS DB Cluster Snapshot %s was not deleted", e.name)
```

Please use the same idea to the other errors below too.